serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -21,8 +21,22 @@ servershell.update_attribute = function(object_id, attribute_id, new_value, mult
     let server = servershell.get_object(object_id);
 
     // Cast new values to corresponding data type if wanted
-    new_value = attribute.type === 'number' ? Number.parseInt(new_value) : new_value;
-    new_value = attribute.type === 'boolean' ? !!new_value : new_value;
+    if (attribute.type === 'number') {
+        if (attribute.multi) {
+            new_value = new_value.map(v => Number.parseInt(v));
```

Comment:
What about floats? Is `'number'` attribute type always just `int`s?

Reply:
Yes we don't support floats right now.